### ISSUE TITLE
Warchief's Blessing is being applied to Alliance players

### ISSUE THREAD

**Issue by VeryRare** [LightsHope/server#999]
Warchief's Blessing world buff Spell ID 16609 is being applied to mind controlled Alliance players.

This buff is intended to be horde only. There exists no evidence of Alliance players having this buff in retail vanilla WoW. 

The spell should contain some logic to make sure that the friendly player is a member of the Horde faction.

**Comment by fitigf15** [comment]
Mind controlled target acts as a pet if you use Gnomish Mind Control Cap.
As far as I have experienced, Horde player's pets get Warchief's Blessing.
Therefor if an Alliance player gets mind controlled, he will be a Horde player's pet, meaning that he can get Warchief's Blessing.

**Comment by TimEsfandiari** [comment]
OP change your name to FunPolice

**Comment by Gropy** [comment]
Is there any evidence that its not supposed to be the way it is at this moment?

**Comment by Zonati** [comment]
>This buff is intended to be horde only. There exists no evidence of Alliance players having this buff in retail vanilla WoW.
This is correct, however that doesn't mean this bug report is justified at all; look at this video: https://www.youtube.com/watch?v=HJq24rRTnnY when the horde player is MC capped by the alliance player, you can clearly see at 0:10 in the video that the horde player gains an alliance pvp symbol next to his name, so this establishes that an MC cap on an alliance player should make them aligned with the horde for the 20 second time period. 
This comes nicely into your next point:
>The spell should contain some logic to make sure that the friendly player is a member of the Horde faction. 
Yes this is 100% correct, this already occurs on said server, alliance players CANNOT gain the buff no matter how close they stand to thrall, but when they are mind controlled by a horde player, they become a member of the horde faction like i've just shown above.
And finally:
>Currently some Alliance players are breaking TOS about cross faction collusion and/or multi-boxing in order to abuse this bug and obtain the buff.
Please read the Elysium TOS before you accuse people of breaking rules, cross faction collusion is only breaking TOS when it is an organized attempt to control world bosses, this behaviour doesn't fall under that whstsoever.

**Comment by Zonati** [comment]
OP check out this video: https://www.youtube.com/watch?v=HJq24rRTnnY and you can clearly see at 0:10 that Mind Control cap gives the person the opposing faction pvp icon, which means that for that time period they are a member of the opposing faction.

Also please read up on the Lights' hope TOS before making comments like that, cross faction collusion only applies to world bosses, see here: https://forum.lightshope.org/viewtopic.php?t=394

> We are removing the rule about cross faction collusion from our Terms of Use but we will be handling extreme cases as per our Fair Use clause. There are pros and cons for keeping this rule in at all or extending it outside of world bosses, so we will have to proceed carefully to best weigh our options. Ideally we would involve the community in this process, but how we would organize that is up in the air given it’s such a nuanced situation. We will continue to monitor the situation and react when we feel it is necessary.

**Comment by VeryRare** [comment]
The buff is from a horde only quest, contains faction lore, and once completed Thrall shouts:

> Be bathed in the power of the Warchief! Drink in his might! **Battle for the glory of the Horde!**

I think it's safe to assume that the quest was meant to celebrate the Horde's victory over defeating Rend and is only meant for players of the horde faction.

There exists no evidence of an Alliance player ever having Warchief's Blessing in retail vanilla so why should we allow it? There's no evidence that they could or couldn't get the buff so in this case you can use common sense to ascertain that Blizzard intended for the buff to only be applied to Horde players.

Is it also true that you can only obtain the buff from the MC cap and not from a Priest's Mind Control spell? 

If you mind control a horde player can a paladin buff him? What about if you use the MC cap? Can a horde player obtain the dragonslayer buff in IF if they are MC'd? Is any of this 'blizzlike'?

In terms of cross faction collusion breaking the ToU I would argue that this constitutes an extreme case and should be discussed within the community to determine if it's breaking ToU.  Regardless, I know that some alliance players are multi-boxing to obtain this buff and that is against the ToU.

**Comment by Zonati** [comment]
Although i understand where you're coming from, from a gameplay perspective you have provided zero evidence that it was impossible for Alliance to get this buff. However evidence has been provided which proves that when an alliance/horde player is mind-controlled by the opposing faction, they will belong to that opposing faction for the duration of the spell. Arguing semantics like "Oh but this buff is to celebrate the victory of the horde, Alliance shouldn't get it" Isn't the sort of evidence which should be used in an attempt to change how something works. Since from a gameplay standpoint it makes complete sense how Alliance can achieve this buff, and it would very much surprise me if it didn't work this way back in Vanilla, because of how mind control works. So in my honest opinion the burden is on you to provide evidence that Blizzard had a specific check in place to ensure that Alliance players who were classified as a member of the horde faction couldn't get this buff, considering though all other Horde aligned Players / NPCS with the PvP tag get this buff if they are in close proximity.

In regards to the ToU, i guess we'll agree to disagree, I don't think it's extreme at all, since this doesn't affect anybody outside of the 40 people raiding.

**Comment by ohganot** [comment]
"Our Terms of Use have been changed slightly in regards to cross-faction collaboration. Anything else is now allowed except tagging a world boss and having the other faction kill it for you." - Istaria

Just because you couldn't find any evidence for alliance players having Warchief's Blessing, doesn't mean it wasn't possible back then. Paladin buffs are also meant for "alliance only", yet Horde players were able to get them back in retail vanilla.

**Comment by dimefox** [comment]
It's not video evidence or a screenshot but fwiw

![image](https://user-images.githubusercontent.com/33604137/34029872-82313758-e11f-11e7-9aad-991cc159c22d.png)

https://web.archive.org/web/20071209180903/http://thottbot.com:80/s16609#v747448

**Comment by ohganot** [comment]
Nice find.

**Comment by Slicy** [comment]
The real bug and question though is when are the cooldowns on WCB and Rallying Cry of the Dragonslayer events gonna be added ?

**Comment by Zonati** [comment]
This is just speculation, but since you are made the players pet / minion when mind controlled (depending on which mind control variant) prehaps for the period of time you are mind controlled, you share the recipients reputation? Since when you are mind controlled, enemy faction NPCs appear 'green' to you, suggesting that you are above neutral when mind controlled.

**Comment by Cabro** [comment]
The MCd player, since effectively acts as a pet, would get the buff also IF the one that MCed him had the required reputation to get the buff in the first place.

**Comment by Sylvestersnr** [comment]
In the patch notes of 2.0.1:
''Players will no longer lose faction when under the influence of "Mind Control" or the Gnomish Mind Control Cap.''

**Comment by Cephel** [comment]
Bump

**Comment by ohganot** [comment]
Seems like a closed case to me based on the TBC patch notes which imply that players indeed changed faction during MC in vanilla.

**Comment by staplerx300** [comment]
FWIW I do remember the reputation thing back in the day. I played undead and couldn't get the buff initially until I did some more quests.
